src/envoy/BUILD

```diff
@@ -49,3 +50,10 @@ pkg_tar(
     package_dir = "/usr/local/bin/",
     tags = ["manual"],
 )
+
+container_image(
+    name = "istio-proxy",
+    base = "@distroless_cc//image",
+    tags = ["manual"],
+    tars = [":envoy_tar"],
+)
```

Comment:
The build target for `.deb` package is `//tools/deb:istio-proxy`, perhaps we could move this to `//tools/docker:istio-proxy` to be a bit more consistent?

Reply:
Made it //tools/docker:envoy since it's not full istio-proxy image yet (needs the agent).